src/schemas/ev42/ev42_synth.h

```diff
@@ -1,26 +1,30 @@
 #pragma once
 #include "ev42_events_generated.h"
 #include <memory>
+#include <random>
 #include <string>
 
 namespace FlatBufs {
 namespace ev42 {
 
-class fb {
+class FlatBufferWrapper {
 public:
-  fb() {}
-  fb(fb &&x) { std::swap(builder, x.builder); }
+  FlatBufferWrapper() {}
+  FlatBufferWrapper(FlatBufferWrapper &&x) { std::swap(builder, x.builder); }
   std::unique_ptr<flatbuffers::FlatBufferBuilder> builder;
   EventMessage const *root();
 };
 
-class synth_impl;
+struct synth_impl {
```

Comment:
SynthImpl?

Reply:
I don't know. I got tired of trying to come up with good names to replace poor ones so I stopped. I am pretty sure that the code in the **\*_synth.\***-files belongs in unit-test code anyways but that is a problem for another day.

Reply:
I suggest changing it to `SynthImpl` and, in absence of a better name suggestion, at least add a doxygen comment to explain it is used to generate test data.